Felienne/hedy
# Fix #1640

**Description**
Fixed a small styling bug that occurred due to an incorrect closing of a <div> which in turn only occurred if a user looked at the achievements while **not** being a teacher. See screenshots below for clarification.

![Schermafbeelding 2022-01-07 om 15 00 57](https://user-images.githubusercontent.com/38155424/148556739-d7c5d9b5-298b-4590-970b-bfcb9196334f.png)

![Schermafbeelding 2022-01-07 om 15 18 34](https://user-images.githubusercontent.com/38155424/148556843-fe634bc4-c45e-40e9-851e-c8aee187f7ed.png)

**Fix for**
This PR fixed #1640.

**How to test**
Navigate to the achievements page while not being a teacher and notice the correct alignment of both the general and hidden banner.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
